api/src/pcapi/routes/public/individual_offers/v1/serialization.py

```diff
@@ -124,15 +126,31 @@ def get_category_fields_model(subcategory: subcategories.Subcategory) -> pydanti
 
 
 class StockBody(serialization.BaseModel):
+    booking_limit_datetime: datetime.datetime | None = pydantic.Field(
+        None,
+        description="The date and time after which the offer can no longer be booked",
```

Comment:
The timezone-aware Date and Time after...
Si ils sont pas attentif à l'exemple ils verront tard qu'il faut une TZ